src/helpers/isRainbowTextAvailable.js

```diff
@@ -1,7 +1,6 @@
 import { UIManager } from 'react-native';
-import isNativeStackAvailable from '../helpers/isNativeStackAvailable';
 
-const useNativeButtonAvailable = () =>
-  isNativeStackAvailable && !!UIManager.getViewManagerConfig('RainbowText');
+const useRainbowTextAvailable = () =>
```

Comment:
can we name it to something not using 'use'? perhaps just `isRainbowTextAvaliable`

Reply:
also, is this something we can turn into a constant or does it need to be a function call?

Reply:
yeah, I don't really remember why it is like this